build-tools-internal/version.properties

```diff
@@ -14,7 +14,7 @@ log4j             = 2.19.0
 slf4j             = 2.0.6
 ecsLogging        = 1.2.0
 jna               = 5.10.0
-netty             = 4.1.94.Final
+netty             = 4.1.107.Final
```

Comment:
Does this affect the netty dependency for repository-azure as well?

Reply:
Yes:

https://github.com/elastic/elasticsearch/blob/b3e5a1f29d80144be4daca4dbcd6cde0a15da5c0/modules/repository-azure/build.gradle#L58-L63

Why do you ask?

Reply:
Nothing really. Just to confirm my understanding. Thanks!

Reply:
Note however that we do still (transitively) depend on older versions of Netty too, because of HDFS.

<img width="1190" alt="image" src="https://github.com/elastic/elasticsearch/assets/5058284/4ca990af-3a95-4aa0-8442-6193ea638150">